jwst/refpix/optimized_convolution.py

```diff
@@ -128,53 +115,52 @@ def apply_conv_kernel(data, kernels, sigreject=4.0):
     kernels_l, kernels_r = kernels
     nchan = len(kernels_l)
 
-    L = data[:, :4]
-    R = data[:, -4:]
+    l = data[:, :4]
+    r = data[:, -4:]
 
     # Find the approximate standard deviations of the reference pixels
     # using an outlier-robust median approach. Mask pixels that differ
     # by more than sigreject sigma from this level.
-    # NOTE: The Median Absolute Deviation (MAD) is calculated as the
+    # NOTE: The Median Absolute Deviation (mad) is calculated as the
     #   median of the absolute differences between data values and their
-    #   median. For normal distribution MAD is equal to 1.48 times the
+    #   median. For normal distribution mad is equal to 1.48 times the
     #   standard deviation but is a more robust estimate of the dispersion
-    #   of data values.The calculation of MAD is straightforward but
-    #   time-consuming, especially if MAD estimates are needed for the
+    #   of data values.The calculation of mad is straightforward but
+    #   time-consuming, especially if mad estimates are needed for the
     #   local environment around every pixel of a large image. The
-    #   calculation is MAD = np.median(np.abs(x-np.median(x))).
+    #   calculation is mad = np.median(np.abs(x-np.median(x))).
```

Comment:
I think this is easier to read if the docstring still uses the caps `MAD`, and perhaps this was a consequence of an auto-find-and-replace.  But if it was intentional and you like it better this way I'm fine with it too